app/views/devices_guidance/how_to_order.html.erb

```diff
@@ -118,7 +118,7 @@
                   Not having DfE settings installed
                 </h4>
                 <p class="app-step-nav__paragraph">
-                  These ‘DfE Standard’ devices are provided with factory settings. Schools, colleges, further education institutions, trusts or LAs should configure them at their own cost before lending them to students. This means installing preferred safeguarding measures and bringing them into an existing device management framework.
+                  These ‘Standard' Windows devices are provided with factory settings. Schools, colleges, further education institutions, trusts or LAs should configure them at their own cost before lending them to students. This means installing preferred safeguarding measures and bringing them into an existing device management framework.
```

Comment:
The quotes around "Standard" don't match

Reply:
Fixed, thanks